new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorRestart.java

```diff
@@ -116,8 +208,6 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
    */
   @Test
   @DisplayName("Restart operator when the domain is rolling after the admin credentials are changed")
```

Comment:
Make sure the Rolling Restart and Operator Restart has been occurring simultaneously.

Reply:
will add the check to see if the last managed server pod is deleted in this existing test...

Reply:
admin server gets terminated first when rolling restart happens..added check for the same